### PR TITLE
enh(poke): display full managed DS resources tree on Poke

### PR DESCRIPTION
**Pull request by fontanierh**
## Description

Easier to see what the user sees 

fixes https://github.com/dust-tt/tasks/issues/404

### CODE REVIEW

File: front/components/ConnectorPermissionsTree.tsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ function PermissionTreeChildren({
   onPermissionUpdate,
   parentIsSelected,
   showExpand,
-  showDocumentModal,
+  displaySource,
```
Comment (PopDaph) [inline front/components/ConnectorPermissionsTree.tsx:54]: NIT: `displayDocumentSource`

File: front/components/ConnectorPermissionsTree.tsx [hunk 2]
```diff
@@ -213,6 +219,8 @@ export function PermissionTree({
   canUpdatePermissions,
   onPermissionUpdate,
   showExpand,
+  displaySourceOverride,
```
Comment (PopDaph) [inline front/components/ConnectorPermissionsTree.tsx:222]: What about duplicating this to have a PermissionTree and a PermissionTreePoké, and `PermissionTreeChildren` would always receive the displaySource and connector permissions?

**Review by PopDaph — APPROVED** [review]
Approving cause it's a nit!

File: front/components/ConnectorPermissionsTree.tsx (hunk 2 above)
Comment (fontanierh) [reply front/components/ConnectorPermissionsTree.tsx]: Thanks for the suggestion, done

File: front/pages/poke/[wId]/data_sources/[name]/index.tsx [hunk 3]
```diff
@@ -382,99 +394,103 @@ const DataSourcePage = ({
...
+                </ContextItem.List>
+                {documents.length == 0 ? (
+                  <div className="mt-10 flex flex-col items-center justify-center text-sm text-gray-500">
+                    <p>Empty</p>
+                  </div>
+                ) : null}
+              </>
+            ) : (
+              <PokePermissionTree
+                owner={owner}
```
Comment (spolu) [inline front/pages/poke/[wId]/data_sources/[name]/index.tsx:489]: permissionFilter should be set to `read` here no?

Comment (fontanierh) [reply front/pages/poke/[wId]/data_sources/[name]/index.tsx:489]: why ?

Comment (spolu) [reply front/pages/poke/[wId]/data_sources/[name]/index.tsx:489]: https://dust4ai.slack.com/archives/C050SM8NSPK/p1707400930486179

**Pull request merged**